components/Buttons/tests/unit/ButtonsThemingTest.swift

```diff
@@ -84,4 +84,29 @@ class ButtonsThemingTest: XCTestCase {
     XCTAssertEqual(buttonShape.bottomRightCorner, shapeScheme.smallComponentShape.bottomRightCorner)
     XCTAssertEqual(buttonShape.bottomLeftCorner, shapeScheme.smallComponentShape.bottomLeftCorner)
   }
+  
+  func testFloatingButtonSecondaryTheme() {
+    // Given
+    let button = MDCFloatingButton()
+    let scheme = MDCContainerScheme()
+    let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+    let typographyScheme = MDCTypographyScheme(defaults: .material201804)
+
+    // When
+    button.applySecondaryTheme(withScheme: scheme)
+    
+    // Then
+    XCTAssertEqual(button.backgroundColor(for: .normal), colorScheme.secondaryColor)
+    XCTAssertEqual(button.imageTintColor(for: .normal), colorScheme.onSecondaryColor)
+    
+    XCTAssertEqual(button.titleFont(for: .normal), typographyScheme.button)
+    
+    let buttonShape = button.shapeGenerator as! MDCRectangleShapeGenerator
```

Comment:
Should this be a guard with an XCTAssertFail if it doesn't work?

Reply:
Addressed.

Reply:
Sure. Addressed in 41da40aaf53833e2d8b33b8c226c9fe30d462460.